backend/src/libs/packages/database/libs/enums/common-table-column.enum.ts

```diff
@@ -0,0 +1,7 @@
+const CommonTableColumn = {
```

Comment:
remove this

Reply:
Oh, I created it to use in `UserColumns `type and `UserWithUserDetailsJoin` types but forgot to do it. Can I save `CommonTableColumn ` and use to create those types to reduce code repeating?

Reply:
answered here https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/46#discussion_r1303912528